src/data-workspace/use-data-value-mutation/optimistically-delete-data-value.js

```diff
@@ -8,13 +8,17 @@ export default async function optimisticallyDeleteDataValue({
     // Cancel any outgoing refetches (so they don't overwrite our optimistic update)
     await queryClient.cancelQueries(dataValueSetQueryKey)
 
-    // Snapshot the previous value
-    const previousDataValueSet = queryClient.getQueryData(dataValueSetQueryKey)
+    // Snapshot the previous value.
+    // This query can be undefined when offline;
+    // provide an empty 'response' for easier optimistic update
+    const previousDataValueSet = queryClient.getQueryData(
+        dataValueSetQueryKey
+    ) ?? { dataValues: [], minMaxValues: [] }
```

Comment:
Now that this is not just a function call anymore but also has a default value, we could extract that into its own helper function used by all optimistically-* functions